pints/_log_priors.py

```diff
@@ -81,15 +92,21 @@ def __init__(self, mean, covariance):
         self._covariance = covariance
         self._dimension = mean.shape[0]
 
-    def dimension(self):
-        """ See :meth:`LogPrior.dimension()`. """
-        return self._dimension
-
     def __call__(self, x):
         return np.log(
             scipy.stats.multivariate_normal.pdf(
                 x, mean=self._mean, cov=self._covariance))
 
+    def dimension(self):
+        """ See :meth:`LogPrior.dimension()`. """
+        return self._dimension
+
+    def sample(self, n=1):
+        """ See :meth:`LogPrior.call()`. """
+        # Note: size=n returns shape (n, d)
+        return np.random.multivariate_normal(
+            self._mean, self._covariance, size=n)
+
 
 class NormalLogPrior(pints.LogPrior):
     """
```

Comment:
Typo I think.

>For example: ``p = NormalPrior(0, 1)`` for a mean of ...

I think it should be `NormalLogPrior(0, 1)` instead.

Reply:
Thanks!